src/components/Card.tsx

```diff
@@ -83,14 +66,73 @@ const Card = ({ cardData }: CardProps) => {
                 }}
               >
                 Add To Collection
-              </button>
+              </FigButton>
             )}
           </div>
           <p>{cardData.flavorText}</p>
-        </figcaption>
-      </figure>
-    </div>
+        </FigCaption>
+      </Figure>
+    </CardContainer>
   )
 }
 
 export default Card
+
+const CardContainer = tw.div`m-8 text-center`
+
+const Figure = styled.figure`
+  ${tw`relative rounded-xl overflow-hidden h-full shadow-bottom-right`}
+  &:hover img {
+    ${tw`motion-safe:translate-x-64`}
+  }
+`
+
+const FigImage = tw.img`
+  rounded-xl 
+  w-64 
+  h-80 
+  transition-all 
+  absolute 
+  object-left
+  transform
+  duration-700
+`
+const FigCaption = tw.figcaption`
+  text-white 
+  bg-gray-800 
+  p-8 
+  top-0 
+  h-80 
+  w-64
+  rounded-lg  
+  shadow-bottom-right
+`
+
+const FigButton = styled.button<{ isAdded?: boolean }>`
+  ${({ isAdded }) => {
+    if (isAdded) {
+      return tw`focus:outline-none 
+      text-white 
```

Comment:
there's a lot of overlap between these cases, I would only define the common classes once and use an if statement for the classes that are different